include/config/item.h

```diff
@@ -8,6 +8,7 @@
 #define I_HEALTH_RECOVERY       GEN_LATEST  // In Gen7+, certain healing items recover a different amount of HP than they used to.
 #define I_SITRUS_BERRY_HEAL     GEN_LATEST  // In Gen4+, Sitrus Berry was changed from healing 30 HP to healing 25% of Max HP.
 #define I_VITAMIN_EV_CAP        GEN_LATEST  // In Gen8, the Vitamins no longer have a cap of 100 EV per stat.
+#define I_RAZOR_CLAW_ON_FIELD   GEN_LATEST  // In PLA (here Gen8+), the Razor Claw can be used as an evolution item on the field.
```

Comment:
This shouldn't be a config. We should allow all evolution held items to be usable like stones now that we have official uses, like how we added Linking Cord evolutions.

Reply:
Sounds good to me. I'll get to it.